uwv_dynamic_model.orogen

```diff
@@ -121,6 +121,6 @@ task_context "VelocityEstimator" do
         max_latency 0.05
     end
 
-    runtime_states :MISSING_TRANSFORMATION
+    exception_states :MISSING_TRANSFORMATION
```

Comment:
Ok, I was thinking it was like a property, at least for a static transformation.